src/vmm_stubs.c

```diff
@@ -0,0 +1,40 @@
+// (c) 2017, 2018 Hannes Mehnert, all rights reserved
+
+#include <caml/mlvalues.h>
+#include <caml/alloc.h>
+#include <caml/memory.h>
+#include <caml/fail.h>
+#include <caml/unixsupport.h>
+
+#include <unistd.h>
+
+CAMLprim value vmm_cpu_count (value unit) {
+  CAMLparam1(unit);
+  int r;
+  r = (int)sysconf(_SC_NPROCESSORS_CONF);
```

Comment:
In my understanding `_SC_NPPROCESSORS_CONF` counts all the configured processors (and some may be offline), for our purposes, is it better to only consider CPU's which are online? `_SC_NPPROCESSORS_ONLN`.

Potentially what is the expected outcome if someone tried to bind a vm to an offline cpu?

Reply:
good point, done in [e2a6e6b](https://github.com/robur-coop/albatross/pull/190/commits/e2a6e6bff9ef08c66206f6d0d0aef7a7c0fa7168)